resources/styles/components/learning-guide.less

```diff
@@ -0,0 +1,198 @@
+@lg-width: 1440px;
```

Comment:
Can you prefix local vars with ... oh, `lg` is learning guide. Can you just use `guide` since `lg` also means large?